compiler/src/iree/compiler/InputConversion/StableHLO/Preprocessing/test/stablehlo_to_stablehlo.mlir

```diff
@@ -462,3 +462,12 @@ func.func @broadcast_iota_sort_slice_incorrect_dims(%in : tensor<16x16x16xf32>)
 
 // CHECK-LABEL: @broadcast_iota_sort_slice_incorrect_dims
 // CHECK-NOT: chlo.top_k
+
+// CHECK-LABEL: @concat_remove_zero_extents
```

Comment:
In lit tests, we usually split test cases.

```suggestion

// -----

// CHECK-LABEL: @concat_remove_zero_extents
```

Reply:
Thanks, added.